scripts/report

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+[[ "${1:-}" != "--help" ]] || {
+  cat <<-EOF
+	Prints which versions of the signer are deployed where and with what arguments.
+
+	Usage: $(basename "$0") [DFX_NETWORK]
```

Comment:
small doubt: can the user provide more than a network? did I understood correctly? If yes, should it be written here?

Reply:
You are right, it should be `[DFX_NETWORK...]`.  Updating....